Update Planning with REX pull (#1695)
theforeman/foreman-documentation: guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

=== Content Deployment

Content deployment is the management of errata and packages on content hosts.

> There are two methods for content deployment on {Project}; the default method is *remote execution*, and the second method is the deprecated *Katello agent*.

Reviewer: ```suggestion
There are two methods for content deployment on {Project}.
The default method is *remote execution*, and the second method is the deprecated *Katello agent*.
```

* *Remote execution* - Remote execution allows installation, update, or removal of packages, the bootstrap of configuration management agents, and the trigger of Puppet runs.
You can configure {Project} to perform remote execution either over SSH (push based) or over MQTT/HTTPS (pull based).

> While {ProjectServer} has remote execution enabled by default, it is disabled by default on {SmartProxyServer}s and content hosts, and has to be manually enabled.

Reviewer: ```suggestion
While remote execution is enabled on {ProjectServer} by default, it is disabled on {SmartProxyServer}s and content hosts.
You must enable it manually.
```

+

> This is the preferred method for content deployment.

Reviewer: Personally, I would place this sentence right after the first sentence of the paragraph.

Author: I like it better in this place, because it stands out and it pairs logically with the note about Kattelo agent deprecation.

> * *Katello agent* - Katello agent is primarily tasked with installation and update of packages.

Reviewer: ```suggestion
* *Katello agent* - Katello agent installs and updates packages.
```

It uses the *goferd* service, which communicates to and from the {ProjectServer}.
It is enabled and started automatically on content hosts after a successful installation of the `katello-agent` package.
+

> Note that the *Katello agent* is deprecated and will be removed in a future {Project} version.

Reviewer: ```suggestion
Note that Katello agent is deprecated and will be removed in a future {Project} release.
```
